scripts/zones/Grand_Palace_of_HuXzoi/mobs/Eo_zdei.lua

```diff
@@ -7,6 +7,7 @@
 -- Animation Sub 3 Ring Form
 -----------------------------------
 
+require("/scripts/zones/Grand_Palace_of_HuXzoi/MobIDs");
```

Comment:
I tend to just strip them as come across them figuring if you have to make another commit for something else anyway, may as well.